README.md

```diff
@@ -62,34 +66,34 @@ the Giver selects two cards from their hand and gives them to the Receiver.
 ## Hidden Trap Card
 _Added by Matt, 14 Aug 2023_
 
-You may not look at your trap card once you have set it.
+A player may not look at their trap card once they have set it.
 
 ## Bartok
 _Added by Mathias, 21 Aug 2023_
 
-If you begin to have 1 card, then you must say "Bartok" before someone next plays a card.
+If a player begins to have 1 card, then they must say "Bartok" before a player next plays a card.
 
 ## Aces Reverse
 _Added by Xavier, 21 Aug 2023_
 
-If you play an Ace, then turn order is reversed.
+If a player plays an Ace, then turn order is reversed.
 
 ## Toktok
 _Added by Jonathan, 21 Aug 2023_
 
-If you begin to have two cards, then you must say "Toktok" before someone next plays a card.
+If a player begins to have two cards, then they must say "Toktok" before a player next plays a card.
 
 ## Delay of Game
 _Added by Matt, 21 Aug 2023_
 
-You may at any point raise your hand to indicate that the current player is taking too long.
-If the majority of players raise their hands, the current player draws a penalty card.
+A player may at any point raise their hand to indicate that the current player is taking too long.
+If the majority of players raise their hands, the Pending Player draws a penalty card.
```

Comment:
Pending or active player

Reply:
Pending. We can be waiting for someone to do something even if it's not their turn.